index.bs

```diff
@@ -2614,26 +2561,6 @@ partial interface MLGraphBuilder {
   </details>
 </div>
 
-#### {{MLGraphBuilder/elu(options)}} #### {#api-mlgraphbuilder-elu-options}
```

Comment:
Nit: when removing section headers for deleted overloads, if there's just one sibling section header then please remove it too.

Reply:
Good catch. Done!